composer.json

```diff
@@ -38,17 +38,23 @@
     },
     "require": {
         "php": "^8.1",
-        "shopware/core": "^6.5.0",
-        "shopware/storefront": "^6.5.0",
```

Comment:
Use old ^6.5.0, since this is working with all future versions like 6.6.*, and this allows us to avoid creating a new plugin release for every new Shopware version.

https://getcomposer.org/doc/articles/versions.md#caret-version-range-